vizro-core/CHANGELOG.md

```diff
@@ -11,6 +11,26 @@ See the fragment files in the [changelog.d directory](https://github.com/mckinse
 
 <!-- scriv-insert-here -->
 
+<a id='changelog-0.1.15'></a>
+
+# 0.1.15 — 2024-04-15
+
+## Highlights ✨
+
+- Add dynamic data, which can be reloaded while the dashboard is running. An optional caching layer enables efficient refreshes with per-data source timeouts. Visit the [user guide on data](https://vizro.readthedocs.io/en/stable/pages/user-guides/data/) for more details. ([#398](https://github.com/mckinsey/vizro/pull/398))
```

Comment:
```suggestion
- Add dynamic data, which can be reloaded while the dashboard is running. An optional caching layer enables efficient refreshes with per data source timeouts. Visit the [user guide on data](https://vizro.readthedocs.io/en/stable/pages/user-guides/data/) for more details. ([#398](https://github.com/mckinsey/vizro/pull/398))
```

Reply:
per-data xxx  seems to be correct though in technical terms: https://stackoverflow.com/questions/10814735/datasource-setlogintimeout-and-drivermanager-setlogintimeout-global

Reply:
As aligned with @maxschulz-COL - we can change it later again, but I'll merge now 👍

Reply:
I would probably go with the hyphen. I think it's because the phrase is used as an adjective but could be wrong about the reason, but pretty sure I'm correct about the usage. Let's drop it in next time.